docs/resources/identity_access_key.md

```diff
@@ -11,10 +11,12 @@ Manages a permanent Access Key resource within HuaweiCloud IAM service.
 ## Example Usage
 
 ```hcl
+variable "user_1_password" {}
+
 resource "huaweicloud_identity_user" "user_1" {
   name        = "user_1"
   description = "A user"
-  password    = "password123!"
+  password    = user_1_password
```

Comment:
also you can just define it with **password**
```suggestion
  password    = var.password
```

Reply:
done